test/import/FSHImporter.ValueSet.test.ts

```diff
@@ -0,0 +1,552 @@
+        * codes from system ZOO where code not-in "#goose"
+        `;
+        const result = importText(input, 'NoGoose.fsh');
+        expect(result.valueSets.size).toBe(1);
+        const valueSet = result.valueSets.get('NoGooseVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(valueSet.components[0], 'ZOO', undefined, [
+          {
+            property: VsProperty.CODE,
+            operator: VsOperator.NOT_IN,
+            value: '#goose'
+          }
+        ]);
+      });
+
+      it('should log an error when the not-in filter has a non-string value', () => {
+        const input = `
+        ValueSet: NoGooseVS
+        * codes from system ZOO where code not-in /duck|goose/
+        `;
+        const result = importText(input, 'NoGoose.fsh');
+        expect(result.valueSets.size).toBe(1);
+        const valueSet = result.valueSets.get('NoGooseVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(valueSet.components[0], 'ZOO', undefined, []);
+        expect(loggerSpy.getLastMessage()).toMatch(/"not-in".*string/);
+        expect(loggerSpy.getLastMessage()).toMatch(/File: NoGoose\.fsh.*Line: 3\D/s);
+      });
+
+      it('should parse a value set that uses filter operator generalizes', () => {
+        const input = `
+        ValueSet: MustelidVS
+        * codes from system ZOO where code generalizes #mustela-nivalis "least weasel"
+        `;
+        const result = importText(input, 'Mustelids.fsh');
+        expect(result.valueSets.size).toBe(1);
+        const valueSet = result.valueSets.get('MustelidVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(valueSet.components[0], 'ZOO', undefined, [
+          {
+            property: VsProperty.CODE,
+            operator: VsOperator.GENERALIZES,
+            value: new FshCode('mustela-nivalis', undefined, 'least weasel')
+              .withLocation([3, 56, 3, 86])
+              .withFile('Mustelids.fsh')
+          }
+        ]);
+      });
+
+      it('should log an error when the generalizes filter has a non-code value', () => {
+        const input = `
+        ValueSet: MustelidVS
+        * codes from system ZOO where code generalizes "least weasel"
+        `;
+        const result = importText(input, 'Mustelids.fsh');
+        expect(result.valueSets.size).toBe(1);
+        const valueSet = result.valueSets.get('MustelidVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(valueSet.components[0], 'ZOO', undefined, []);
+        expect(loggerSpy.getLastMessage()).toMatch(/"generalizes".*code/);
+        expect(loggerSpy.getLastMessage()).toMatch(/File: Mustelids\.fsh.*Line: 3\D/s);
+      });
+
+      it('should parse a value set that uses filter operator exists', () => {
+        const input = `
+        ValueSet: ZooVS
+        * codes from system ZOO where display exists true
```

Comment:
This is a minor comment, so if this is difficult to implement, I would say really don't worry about it for now. That said, I find this wording a little bit odd. Maybe it would make sense to do something like:
`where display exists` or `where display does not exist` instead of `where display exists true` or `where display exists false`. 
But trust me, I (and @jafeltra) have done plenty of work that has to deal with the complexities of English grammar before, so the added code to make something like that work is honestly maybe not worthwhile.

Reply:
I like the idea that `where display exists` should be allowed as an equivalent to `where display exists true`, and it should introduce minimal complications, so I'll try to add it. Having the special grammar for `does not exist` as an equivalent to `where display exists false` would be nice, but would more or less need to be treated as a special case in the grammar, since it doesn't match the current rule `vsFilterOperator: EQUAL | SEQUENCE`.

Reply:
Okay, that sounds good! Don't worry about `does not exist`, then. But if it doesn't take much time to do `where display exists` then that would be nice.